build.gradle

```diff
@@ -67,5 +67,5 @@ ext {
   androidMapsUtilsVersion = "0.5+"
   androidx = "1.0.0"
   constraintLayout = "1.1.3"
-  threetenbp = "1.3.8"
+  threeTenBp = "1.3.8"
```

Comment:
Shouldn't we be 1.1.2?

Reply:
Well, this code is now outdated. It used to use the ThreeTenBP instead of the ThreeTenABP which was in a higher version. Should be fine now.